pkg/controller/plan/adapter/ocp/validator.go

```diff
@@ -84,39 +84,44 @@ func (r *Validator) StorageMapped(vmRef ref.Ref) (ok bool, err error) {
 	}
 
 	for _, vol := range vm.Spec.Template.Spec.Volumes {
+		var pvcName string
 		switch {
 		case vol.PersistentVolumeClaim != nil:
-			// Get PVC
-			pvc := &core.PersistentVolumeClaim{}
-			err = r.sourceClient.Get(context.TODO(), k8sclient.ObjectKey{
-				Namespace: vmRef.Namespace,
-				Name:      vol.PersistentVolumeClaim.ClaimName,
-			}, pvc)
-			if err != nil {
-				err = liberr.Wrap(
-					err,
-					"PVC not found.",
-					"pvc",
-					vol.PersistentVolumeClaim.ClaimName)
-				return
-			}
+			pvcName = vol.PersistentVolumeClaim.ClaimName
+		case vol.DataVolume != nil:
+			pvcName = vol.DataVolume.Name
+		default:
+			continue
```

Comment:
shall we add a log that indicates we skip the volume here?

Reply:
added